examples/mlflow_deployment/README.md

```diff
@@ -142,7 +142,7 @@ def predictor(
 ) -> Output(predictions=np.ndarray):
     """Run a inference request against a prediction service"""
 
-    service.start(timeout=10)  # should be a NOP if already started
+    service.start(timeout=60)  # should be a NOP if already started
```

Comment:
I'm wondering why we even need to supply a custom timeout value here, doesn't that indicate that our default value is somehow wrong?

Reply:
We've already made this change in a separate PR (#1294). I just needed it in here as well for the example to work while working on the code.

Reply:
#1308 is just waiting for a review before it gets merged in. Same changes here...